src/core/utils/helpers/general.ts

```diff
@@ -391,7 +391,7 @@ export const getAllFaustIds = (manifestations: Manifestation[]) => {
 export const getScrollClass = (modalIds: string[]) => {
   return modalIds.length > 0 ? "scroll-lock-background" : "";
 };
-export const dataIsNotEmpty = (data: unknown[]) => Boolean(data.length);
+export const isDataEmpty = (data: unknown[]) => Boolean(!data.length);
```

Comment:
Consider replacing `isDataEmpty()` with [`_.isEmpty()`](https://lodash.com/docs/4.17.15#isEmpty) since we are already committed to Lodash.